### ISSUE TITLE
Use JsTrans with yiic message

### ISSUE THREAD

**Issue by KirillRyzhkov** [arne-s/JsTrans#12]
Can you please point me to the right direction? I have a Yii project and translation is generated using yiic. I am able to generate string with Yii.t translator. 
In JS files I have alert(Yii.t("translate","Your message has been sent successfully"));

I installed your extension this way
Yii::setPathOfAlias('extensions', dirname(__FILE__).'/../../../extensions');

'import'=>array(
		'application.models.*',
		'application.components.*',
		'extensions.JsTrans.*',
	),

'jsTrans'=>array(
'class'=>'extensions.JsTrans.JsTrans',
'languages'=>array('en','es'), // the languages to be made available
),

So the confusing part for me is this line in the README file
new JsTrans('app','nl');

Where do I need to put this line of code to work correctly with yiic?

**Comment by bora89** [comment]
```PHP
new JsTrans('app','nl');
```
It is actual "publishing" translations into page. Consider that line as "include" statement. I include this in init() method of widget which has to have JS transtation avalable.

**Comment by KirillRyzhkov** [comment]
Thanks for the answer. Still this part is a bit confusing for me "include this in init() method of widget which has to have JS transtation avalable".

I have javascript files in site/js folder so as I understood correctly I should uncomment this section:
/****
 // optionally to make jsTans available everywhere, without having to call 
 // Yii::app()->jsTrans before using it
 'preload'=>array('jsTrans', ... ),
*/
and put in the config file or main layout page and what should I have in array('jsTrans', ... ) in my situation?

Help is really appreciated because it is the only option for me to translate messages in javascript files.

**Comment by bora89** [comment]
Yes, I also have this preload option, I created translation file which should be available on accross all pages, named it like "common.php" and used with 'preload'=>array('jsTrans', ... ), as you showed.
So, philosophy is this:
Either you preload with config for all pages,
OR you make "new JsTrans('app','nl')" calls only in places where it is required.
In my opinion it is clear enough, but I agree that "new JsTrans('app','nl')" mislead somehow.

**Comment by KirillRyzhkov** [comment]
Ok, thanks for help
